src/bin/signer-process-fail.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2024 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License,
+// Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+// or the MIT license (http://opensource.org/licenses/MIT),
+// at your option.
+// Unless required by applicable law or agreed to in writing,
+// this software is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR REPRESENTATIONS OF ANY KIND, either express or
+// implied. See the LICENSE-MIT and LICENSE-APACHE files for the
+// specific language governing permissions and limitations under
+// each license.
+
+fn main() {
```

Comment:
It would would be a good idea to have the example write an error to stderr so there's some kind of reporting in this case.

Reply:
Will do, thanks!

Reply:
Resolved here: https://github.com/contentauth/c2patool/pull/169/commits/86bebfc451e75be1addf406c564748aeb5905616